cli_collide.py

```diff
@@ -1,10 +1,14 @@
 ''' Command-Line Tool to auto-add ground, BBs, crystals, and skell reefs to a level. 
```

Comment:
`''' `
`Command-Line Tool to auto-add ground, BBs, crystals, and skell reefs to a level. `

Let's put the first line down, so that it's not "indented"